src/test/resources/com/jetbrains/snakecharm/cucumber/resolve/snakemake_params_in_shell.feature

```diff
@@ -31,9 +31,14 @@ Feature: Resolve for params in shell section
         output: "path/to/output"
         params:
           xmx=os
-        shell: "command {params.xmx.path}"
+        shell: "command {params.xmx<suffix>}"
       """
       When I put the caret after {params.xm
       Then reference should resolve to "xmx" in "foo.smk"
+      Examples:
+        | suffix   |
+        | .path    |
+        | [path]   |
```

Comment:
@sharkovadarya Regexp in reference impl doesn't work for 2nd and 3rd cases. Please update regexp to accept identifier (e.g. a sequence of word letters) and then some non-word letter.